jetty-core/jetty-server/src/main/config/modules/eager-content.mod

```diff
@@ -0,0 +1,68 @@
+[description]
+Applies the EagerContentHandler to the entire server.
+The EagerContentHandler can eagerly load content asynchronously before calling the next handler.
+Typically, this handler is deployed before an application that uses blocking IO to read the request body
+and if deployed after this handler, the application will never (or rarely) block for request content.
+This gives many of the benefits of asynchronous IO without the need to write an asynchronous application.
+
+[tags]
+server
+
+[depend]
+server
+
+[after]
+compression
+cross-origin
+gzip
+rewrite
+size-limit
+
+[before]
+qos
+thread-limit
+
+[xml]
+etc/jetty-eager-content.xml
+
+[ini-template]
+#tag::documentation[]
+## The maximum number of form fields or -1 for a default.
+# jetty.eager.form.maxFields=-1
+
+## The maximum size of the form in bytes -1 for a default.
+# jetty.eager.form.maxLength=-1
+
+## The directory where MultiPart parts will be saved as files.
+# jetty.eager.multipart.location=/tmp
+
+## The maximum number of parts that can be parsed from the MultiPart content, or -1 for unlimited.
+# jetty.eager.multipart.maxParts=100
```

Comment:
Still a bit weird that you have -1 for default above and -1 for unlimited for these ones.

I do prefer -1 being unlimited not default, you said you can just set `MAX_VALUE` for unlimited, but that is a bit more annoying because you don't know from here if its an int or a long.